.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
 
       - name: Install pip and setuptools
         run: |
-          python -m pip install --upgrade pip
+          python -m pip install --upgrade pip==23.0.1
```

Comment:
This should not be pinned!

Reply:
Every version of pip past 23.0.1 will break all of our builds, here and on biosimulations.  We need to fix that, but it's a much larger project.